src/app/tests/suites/credentials/TestHarnessDACProvider.cpp

```diff
@@ -196,14 +196,14 @@ TestHarnessDACProvider::TestHarnessDACProvider()
 
 void TestHarnessDACProvider::Init(const char * filepath)
 {
-    constexpr const char kDacCertKey[]      = "dac_cert";
-    constexpr const char kDacPrivateKey[]   = "dac_private_key";
-    constexpr const char kDacPublicKey[]    = "dac_public_key";
-    constexpr const char kPaiCertKey[]      = "pai_cert";
-    constexpr const char kCertDecKey[]      = "certification_declaration";
-    constexpr const char kFirmwareInfoKey[] = "firmware_information";
-    constexpr const char kIsSuccessKey[]    = "is_success_case";
-    constexpr const char kDescription[]     = "description";
+    static constexpr char kDacCertKey[]      = "dac_cert";
```

Comment:
Out of curiosity, does `static` change the generated code anyhow when applied to `constexpr` char array?

Reply:
Tried it on a trivial test on godbolt.org, the result seemed quite the same, however assuming this was due to scripting and it dose not seem wrong. Marking this resolved to get this to merge.

Reply:
This is done to ensure we're not initializing the the array at runtime (can't speak to whether optimizing this is guaranteed to succeed in the absence of 'static').

Reply:
Note that this would be undefined behavior (and trigger the diagnostic -Wreturn-stack-address)

```
const char *GetString() {
  constexpr char kStringConstant[] = "foo";
  return kStringConstant;
}
```

We don't have that lifetime issue with const char * since it's a pointer to a literal string which is always "static".